spec/app2_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'chefspec'
+require 'chefspec/berkshelf'
+
+describe 'osl-app::app1' do
+  context 'on centos 7.2' do
+    let(:chef_run) do
+      ChefSpec::SoloRunner.new(platform: 'centos',
+                               version: '7.2.1511').converge(described_recipe)
+    end
+
+#    before do
+#      stub_data_bag_item('osl-app', 'iam').and_return(
+#        secret_key_base: '7eef5c70ecb083192f46e601144f9d77c9b66061b634963a507'\
+#          '0fb086ae78bc9353af2c6311edb168abbb9d0bd428f800a0b1713534cf4ad239e8d'\
+#          '07fdd16c34'
+#      )
+#    end
```

Comment:
Unless you're going to be using an encrypted data bag for any reason, I'd just go ahead and remove this.